test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -83,6 +83,8 @@ build_branch()
         . ./test/evergreen/find_cmake.sh
         config=""
         config+="-DENABLE_SNAPPY=1 "
+	# Need to disable configs since WT-9455
```

Comment:
nitpick, indentation here seems a bit weird.

Reply:
It was a tab :(, fixed.

Reply:
Thanks, the line below it is still an issue though.

Reply:
Done.

Reply:
Thanks!